tests/src/log4j/log4j/1.2.17/src/test/java/log4j/log4j/Log4jTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package log4j.log4j;
+
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.apache.log4j.xml.DOMConfigurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class Log4jTest {
+
+    private ByteArrayOutputStream stdOut;
+    private final PrintStream systemOut = System.out;
+
+    @BeforeEach
+    public void setUp() {
+        stdOut = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(stdOut));
+    }
+
+    @Test
+    void consoleInfoLogger() {
```

Comment:
A few more things: The `consoleInfoLogger` and `consoleInfoLoggerProperties` tests create the `logs` folder and the following files in it:
```
dailyRollingAppender.log
fileAppender.log
rollingFileAppender.log
```
Can you check at the end of each test that the expected message exist in those files, something like:
```
    String loggedMessage = Files.readAllLines(Paths.get(filePath)).get(0);
    assertThat(loggedMessage).contains("info message ");
```
and delete the `logs` folder after each executed test. The `deletion` code can be added to the following method
```
    @AfterEach
    public void tearDown() {
        System.setOut(systemOut);
    }
```